src/OneExplorer.ts

```diff
@@ -226,30 +227,66 @@ export class OneExplorer {
 // menu handler
 //
 
-export {handleRunOnecc};
+import {EventEmitter} from 'events';
 
-/**
- * Function called when onevscode.run-cfg is called (when user clicks 'Run' on cfg file).
- * @param cfgUri uri of cfg file
- */
-function handleRunOnecc(cfgUri: vscode.Uri, logger: Logger) {
-  const toolRunner = new ToolRunner(logger);
-
-  // TODO Refine later
-  const resolve = function(value: string) {
-    console.log('Running onecc was successful!');
-  };
-  const reject = function(value: string) {
-    console.log('Running onecc failed!');
-  };
-
-  const toolArgs = new ToolArgs('-C', cfgUri.fsPath);
-  const cwd = path.dirname(cfgUri.fsPath);
-  let oneccPath = toolRunner.getOneccPath();
-  if (oneccPath === undefined) {
-    throw new Error('Cannot find installed onecc');
-  }
-
-  const runner = toolRunner.getRunner('onecc', oneccPath, toolArgs, cwd);
-  runner.then(resolve).catch(reject);
+class OneccRunner extends EventEmitter {
+  private startRunningOnecc: string = 'START_RUNNING_ONECC';
+  private finishedRunningOnecc: string = 'FINISHED_RUNNING_ONECC';
+
+  constructor(private cfgUri: vscode.Uri, private logger: Logger) {
+    super();
+  }
+
+  /**
+   * Function called when onevscode.run-cfg is called (when user clicks 'Run' on cfg file).
+   */
+  public run() {
+    const toolRunner = new ToolRunner(this.logger);
+
+    this.on(this.startRunningOnecc, this.onStartRunningOnecc);
+    this.on(this.finishedRunningOnecc, this.onFinishedRunningOnecc);
+
+    const toolArgs = new ToolArgs('-C', this.cfgUri.fsPath);
+    const cwd = path.dirname(this.cfgUri.fsPath);
+    let oneccPath = toolRunner.getOneccPath();
+    if (oneccPath === undefined) {
+      throw new Error('Cannot find installed onecc');
```

Comment:
(optional)
I'd be better to let user know about detail

(1) onecc is not given in cfg file
(2) The installed onecc is found in the system
(2-1) if onecc is not given in cfg file, so getOneccPath() tried to find default path ('/usr/share/one/bin/onecc') and it doens't exist.
(2-2) given onecc is not found

Reply:
Please ignore the message above